client/src/charts/canada/CanadaD3Component.js

```diff
@@ -153,14 +150,11 @@ export class CanadaD3Component {
     };
 
     // Set province colours, attach event dispatchers
-    const province_is_active = (prov_key) =>
-      _.some(data, (year) => year[prov_key]);
+    const province_is_active = (prov_key) => data[prov_key];
```

Comment:
On a separate note

![image](https://user-images.githubusercontent.com/25855114/90545004-fc532a80-e155-11ea-9ea8-647a1d76446c.png)

Ontario seems to have lost its label while nova scotia's hangs around

![image](https://user-images.githubusercontent.com/25855114/90545092-1b51bc80-e156-11ea-910b-27acf6653c38.png)